app/addons/replication/api.js

```diff
@@ -295,10 +297,16 @@ export const combineDocsAndScheduler = (docs, schedulerDocs) => {
   });
 };
 
-export const fetchReplicationDocs = () => {
+export const fetchReplicationDocs = ({docsPerPage, page}) => {
+  const params = {
+    limit: docsPerPage + 1,
+    skip: (page - 1) * docsPerPage,
+    include_docs: true
+  };
+
   return supportNewApi()
     .then(newApi => {
-      const docsPromise = get('/_replicator/_all_docs?include_docs=true&limit=100')
+      const docsPromise = get(`/_replicator/_all_docs?startkey=%22_designZ%22&${app.utils.queryParams(params)}`)
```

Comment:
Can I know what is this suppose to do? When I run locally your branch, I don't have anything in my replicator activity and I suspect this line to the be the cause.

Reply:
This fetches all the docs in the replicator db except the design doc.

Reply:
I agree with @popojargo - because of the `startkey` param, the query won't return any docs whose ID starts with a number or uppercased letter.

Reply:
The only viable solution I see is splitting the request in two API calls. One to fetch documents before the _design (?endkey="_design*") and another one to fetch documents after design docs (startkey="_designZ"